transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/Dialect.java

```diff
@@ -131,4 +133,64 @@ public interface Dialect {
                 }
               })
           .build();
+
+  Dialect MS_SQL_SERVER =
```

Comment:
Given that this is a completely new implementation it would be nicer to just start from the latest migration id and create everything in the start state in one go, rather than replicating the full upgrade chain from when this library was first created...

Reply:
Should be addressed.